src/main/java/org/mifos/connector/channel/camel/routes/ChannelRouteBuilder.java

```diff
@@ -247,6 +247,28 @@ public void configure() {
                     exchange.getIn().setBody(response.toString());
                 });
 
+        /*
+         * Tester endpoint for starting mpesa workflow
+         */
+        from("rest:POST:/channel/mpesa/transaction")
```

Comment:
this is just a testing endpoint, since we need to test the mpesa workflow, so I created this for starting the workflow. 
We already discussed it that `transactionType`(checkout bellow json) will be the deciding factor for either it is a mpesa request or not, so either way it will be removed once testing is done. 

```json
 {
    "scenario": "MPESA",
    “subType”: “BUYGOODS”,
    "initiator": "PAYEE",
    "initiatorType": "BUSINESS"
  }

```

Reply:
As discussed, please change it to /channel/collection.

Reply:
I have renamed it